Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM ghcr.io/celestiaorg/celestia-node:v0.8.2 AS celestia-node
+FROM ghcr.io/celestiaorg/celestia-app:0.12.2 AS celestia-app
 
-FROM ghcr.io/celestiaorg/celestia-app:0.12.2
+FROM ghcr.io/celestiaorg/celestia-node:v0.8.2
 
-COPY --from=celestia-node /bin/celestia /
+USER root
```

Comment:
Then why do we not follow the same principles for this refactoring PR? 
https://github.com/celestiaorg/celestia-node/blob/main/Dockerfile#L21-L22

Reply:
It has `celestia-node` as a parent image. See [here](https://github.com/rollkit/local-celestia-devnet/pull/22/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R3).
Therefore it builds on that container, which has the celestia user setup. We changed to `root` user to install the needed packages, as the `celestia` user has no permission to do so. Then we change back to the `celestia` user; see [here](https://github.com/rollkit/local-celestia-devnet/pull/22/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R13).

Reply:
Oh I see. Thanks! 
Left 1 nit